package.json

```diff
@@ -21,7 +21,7 @@
   "dependencies": {
     "@types/elliptic": "^6.4.6",
     "@types/events": "^1.2.0",
-    "@types/ipfs": "git+https://github.com/lukas2005/types-ipfs.git",
-    "orbit-db": "git+https://github.com/orbitdb/orbit-db.git"
+    "ipfs-core-types": "^0.10.1",
+    "orbit-db": "orbitdb/orbit-db"
```

Comment:
how does this work? is this abbreviated github?

Reply:
https://docs.npmjs.com/cli/v8/configuring-npm/package-json#github-urls

both should work i think. for some reason i changed it to this (git+url seemed slower)